setup.py

```diff
@@ -81,13 +81,23 @@ def get_version(version_file):
     ],
     python_requires=">=3.6",
     # What does your project relate to?
-    keywords="skeleton",
+    keywords="posture and exposure report",
     packages=find_packages(where="src"),
     package_dir={"": "src"},
-    package_data={"example": ["data/*.txt"]},
+    package_data={"pe_reports": ["data/*.txt"]},
     py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
     include_package_data=True,
-    install_requires=["docopt", "schema", "setuptools >= 24.2.0"],
+    install_requires=[
+        "docopt",
+        "schema",
+        "setuptools >= 24.2.0",
+        "openpyxl",
+        "pymongo",
+        "pymupdf",
+        "docopt",
+        "glob2",
+        "pandas",
+    ],
```

Comment:
Please sort this list of requirements alphabetically.

Reply:
Resolved in e694f6d and e5b92f2